ddtrace/contrib/pylons/middleware.py

```diff
@@ -48,6 +49,8 @@ def __call__(self, environ, start_response):
             # Set the service in tracer.trace() as priority sampling requires it to be
             # set as early as possible when different services share one single agent.
             span.span_type = http.TYPE
+            # Send to Trace Search. TODO: plug it to ddtrace.config.
+            span.set_tag(EVENT_SAMPLE_RATE_KEY, 1)
```

Comment:
Just want to make sure that we're all fine with enabling this by default. Will there be any adverse effects for non-trace-search users (doubt it, but wanna be sure)?